.github/workflows/cli.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
           go-version: 1.20.6
       - uses: engineerd/setup-kind@v0.5.0
         with:
-          version: "v0.17.0"
+          version: "v0.20.0"
```

Comment:
I'm plan to update this part to Makefile or hack/shell for reduce fragmentation.

Reply:
Do you mean get rid of this `engineerd/setup-kind`? And install kind by Makefile or shell?

Reply:
I missed it during https://github.com/karmada-io/karmada/pull/3913.